### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by ARamsey118**

### CODE REVIEW

File: packages/desktop/i3/config/i3.config [hunk 1]
```diff
@@ -35,8 +35,10 @@ client.unfocused 			$background $background $foreground $background
 hide_edge_borders none # hiding borders
 new_window pixel 4
 
-gaps inner 12 # i3 gaps
-gaps outer 3
+#set $default_gaps_inner 12
```
Comment (Serubin) [inline packages/desktop/i3/config/i3.config]: Commented out?

File: bash/.prompt [hunk 2]
```diff
@@ -40,7 +40,7 @@ if [ "$EUID" -ne 0 ]; then
 fi
 
 
-export PS1="$BBlue$User@$HostName$BWhite$Color_Off:$BCyan$PathShort$BWhite"'$(git branch &>/dev/null;\
+export PS1="$BCyan$User@$HostName$BWhite$Color_Off:$BCyan$PathShort$BWhite"'$(git branch &>/dev/null;\
```
Comment (Serubin) [inline bash/.prompt]: Going to be picky - I like the color separation.

File: bash/.env [hunk 3]
```diff
@@ -32,4 +36,8 @@ shopt -s checkwinsize
 export HISTCONTROL=ignoredups:erasedups
 
 export CLICOLOR=1 # for ls colors
+export EDITOR=vim
 
+#export TERM="screen-256color" #tmux colors
+[[ $TMUX = "" ]] && export TERM="xterm-256color"
```
Comment (Serubin) [inline bash/.env]: This is already done in the tmux config. No need for it to be set here

File: bash/.custom [hunk 4]
```diff
@@ -1,3 +0,0 @@
-#!/bin/bash
```
Comment (Serubin) [inline bash/.custom:1]: Please

Please :)

File: packages/cli/git/config/.gitconfig [hunk 5]
```diff
@@ -22,3 +22,5 @@
 	tool = opendiff
 [help]
 	autocorrect = 1
+[credential]
+	helper = /usr/lib/git-core/git-credential-gnome-keyring
```
Comment (Serubin) [inline packages/cli/git/config/.gitconfig]: What does this do?

Comment (ARamsey118) [inline packages/cli/git/config/.gitconfig]: It makes gnome keyring store git credentials. I'm taking it out because ssh keys are far better

**Pull request merged**